### PR TITLE
WHO 7 cards

### PR DESCRIPTION
**Pull request by Simisays**
My god these cards are wordy !

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/blink.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Blink
+ManaCost:2 U B
+Types:Enchantment Saga
+K:Saga:4:DBShuffle,DBToken,DBShuffle,DBToken
+SVar:DBShuffle:DB$ ChangeZone | ValidTgts$ Creature | Origin$ Battlefield | Destination$ Library | Shuffle$ True | RememberChanged$ True | SubAbility$ DBInvestigate | SpellDescription$ Choose target creature. Its owner shuffles it into their library, then investigates. (They create a Clue token.)
+SVar:DBInvestigate:DB$ Investigate | Num$ 1 | Defined$ RememberedOwner | SubAbility$ DBCleanup | StackDescription$ {p:TargetedController} investigates.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/blink.txt:6]: StackDescription useless?

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/blink.txt:6]: a zone change can't lead to a different owner, so remembering isn't needed

File: forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Day of the Moon
+ManaCost:2 R
+Types:Enchantment Saga
+K:Saga:3:DBNameCard,DBNameCard,DBNameCard
+SVar:DBNameCard:DB$ NameCard | Defined$ You | ValidCards$ Card.Creature | ValidDesc$ Creature | SubAbility$ DBRemember | SpellDescription$ Choose a creature card name, then goad all creatures with a name chosen for Day of the Moon. (Until your next turn, they attack each combat if able and attack a player other than you if able.)
+SVar:DBRemember:DB$ Pump | RememberObjects$ ValidAll Card.NamedCard | SubAbility$ DBGoadAll
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt:6]: this doesn't really work, creatures can change names

File: forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Trenzalore Clocktower
+ManaCost:no cost
+Types:Legendary Land
+A:AB$ Mana | Cost$ T | Produced$ R | SubAbility$ DBPutCounter | SpellDescription$ Add {U}.
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ TIME | SpellDescription$ Put a time counter on CARDNAME.
+A:AB$ ChangeZoneAll | Cost$ 1 U T SubCounter<12/TIME> Exile<1/CARDNAME> | IsPresent$ Time Lord.YouCtrl | Origin$ Graveyard,Hand | Destination$ Library | ChangeType$ Card.YouOwn | Shuffle$ True | SubAbility$ DBDraw | SpellDescription$ Shuffle your graveyard and hand into your library, then draw seven cards. Activate only if you control a Time Lord.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: do you mean you can't activate it?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: Yeah, I guess it has something to do with the Time Lord thing ?

File: forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt (hunk 2 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt:6]: Hmm I used null chamber a template but I guess that's not possible here. 

Is this scriptable without engine changes ?

File: forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt (hunk 3 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: yeah the space for "Time Lord" in the Present Check might tip it off

@tool4ever you probably need to debug that?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: I think `CardType.parse()` does not work with it in some cases
but I don't know when/if I have time to fix it yet

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: Try rebase and see if it works now

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/trenzalore_clocktower.txt:6]: Yes thanks 
image

File: forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt (hunk 2 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt:6]: probably something for @Northmoc

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt:6]: Looks neat, I can take a stab at it

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/day_of_the_moon.txt:6]: I'll remove this one from my PR  then

**Review by tool4ever — APPROVED** [review]

**Pull request merged**